pkg/autoscaler/config.go

```diff
@@ -48,10 +48,7 @@ type Config struct {
 	PanicWindow    time.Duration
 	TickInterval   time.Duration
 
-	ScaleToZeroThreshold   time.Duration
```

Comment:
This is a nice simplification.  But now that the stable window and scale-to-zero threshold are one and the same, we need to increase the stable window.  1 minute is not enough to allow for Istio route propagation.  And we want to make sure that the stable window is at least 1 minute *larger* than the grace period (so we don't *always* scale to zero, even with traffic.)

How about a stable window of 2 minutes and a grace period of 1 minute?  @mattmoor do you have opinions about these values?

Reply:
Hm, the grace period stays the same and the grace period is responsible to catch istio route propagation though, isn't it? The KPA will take a `0` from the autoscaler to setup the routes and only after the grace period (which is still 2 minutes) it will actually scale down to 0.